packages/mobile/src/geth/networkConfig.ts

```diff
@@ -101,7 +101,7 @@ const networkConfigs: { [testnet: string]: NetworkConfig } = {
     syncMode: DEFAULT_SYNC_MODE,
     initiallyForno: FORNO_ENABLED_INITIALLY,
     blockchainApiUrl: 'https://blockchain-api-dot-celo-mobile-mainnet.appspot.com/',
-    odisUrl: 'https://us-central1-celo-odis-mainnet.cloudfunctions.net',
+    odisUrl: 'https://us-central1-celo-pgpnp-mainnet.cloudfunctions.net/',
```

Comment:
Should we open an issue to track hosting this under a new name?

Reply:
Will do

Reply:
https://github.com/celo-org/celo-monorepo/issues/4773